### PR TITLE
Make GLExtensions no longer require slghtly modified GL headers

### PR DESCRIPTION
**Pull request by Sonicadvance1**
In fact, removes the need for external headers at all.

### CODE REVIEW

File: Source/Core/Common/GL/GLExtensions/AMD_pinned_memory.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2013 Dolphin Emulator Project
```
Comment (ZephyrSurfer) [inline Source/Core/Common/GL/GLExtensions/AMD_pinned_memory.h]: I thought copyright year was to be set as the year the file was created?

File: Source/Core/Common/GL/GLExtensions/gl_common.h [hunk 2]
```diff
@@ -2,6 +2,166 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include <GL/gl.h>
-#include <GL/glext.h>
+/**********************************************************************
+ * Begin system-specific stuff. Do not do any of this when building
+ * for SciTech SNAP, as this is all done before this header file is
```
Comment (delroth) [inline Source/Core/Common/GL/GLExtensions/gl_common.h]: Uh?

And you're slapping the Dolphin copyright on this?

**Comment by delroth** [comment]
Please resolve the ownership/licensing issues. We can't merge this when it's clearly taken from another project without keeping original authorship information.

**Comment by Sonicadvance1** [comment]
Oops. I had forgotten the licensing stuff in my copy and paste spree

**Pull request merged**